src/Sql/Sql/ChangeLog.md

```diff
@@ -28,6 +28,8 @@
 * Added a new cmdlet for Azure SQL Managed Instance refresh external governance status
   - 'Invoke-AzSqlInstanceExternalGovernanceStatusRefresh'
 * Updated `Get-AzSqlInstance` to support returning the `ExternalGovernanceStatus` property
+* Added multi-secondary support for `Get-AzSqlDatabaseFailoverGroup`, `Remove-AzSqlDatabaseFromFailoverGroup` and `Add-AzSqlDatabaseFromFailoverGroup`
```

Comment:
Move it to "## Upcoming Release"

Reply:
Good catch, bumped it back up to upcoming release in latest revision.